sklearn/metrics/tests/test_common.py

```diff
@@ -1064,3 +1064,26 @@ def test_sample_weight_invariance(n_samples=50):
         else:
             yield (check_sample_weight_invariance, name, metric, y_true,
                    y_pred)
+
+
+def test_no_averaging_labels():
+    # test labels argument when not using averaging
+    # in multi-class and multi-label cases
+    y_true_multilabel = np.array([[1, 1, 0, 0], [1, 1, 0, 0]])
```

Comment:
Why are you referencing label `4` when the only labels available are `0, 1, 2, 3`? It's a wonder that this works... Or rather, it's no surprise this breaks #4287...